docs/flatcar-linux/aws.md

```diff
@@ -109,7 +109,7 @@ module "aws-tempest" {
 
   # configuration
   ssh_authorized_key = "ssh-rsa AAAAB3Nz..."
-  asset_dir          = "/home/user/.secrets/clusters/tempest"
+  asset_dir          = "./assets"
```

Comment:
This will break this line: https://github.com/kinvolk/lokomotive-kubernetes/pull/82/files#diff-d8962d6ba4490833047bc8e884ed9d23R165

Idem for all the other docs

Reply:
nice catch, fixed it.